src/Symfony/Component/VarDumper/VarDumper.php

```diff
@@ -37,6 +38,26 @@ public static function dump($var)
             }
 
             self::$handler = function ($var) use ($cloner, $dumper) {
+                (function () {
+                    list('name' => $name, 'file' => $file, 'line' => $line) = (new SourceContextProvider())->getContext();
+
+                    $attr = array();
+                    if ($this instanceof HtmlDumper) {
+                        if (\is_string($file)) {
+                            $attr = array(
+                              'file' => $file,
+                              'line' => $line,
+                              'title' => $file,
+                            );
+                        }
+                    } else {
+                        $name = $file;
+                    }
+
+                    $this->line = $this->style('meta', $name, $attr).' on line '.$this->style('meta', $line).':';
```

Comment:
In this scenario, we are 100% sure that the dumper is an instance of `CliDumper` so those methods are safe to use.